src/service/viewer-impl.js

```diff
@@ -802,6 +802,24 @@ export class Viewer {
   }
 
   /**
+   * Get the fragment from the url or the viewer
```

Comment:
Missing trailing '.'. Also, you want to mention the fragment has the '#' prefix stripped off.